src/components/Hero.tsx

```diff
@@ -5,7 +5,7 @@ import { FaTelegramPlane } from "react-icons/fa"
 import { CSSTransition, SwitchTransition } from "react-transition-group"
 
 const AssetShowcase = () => {
-  const assets = ["AAPL", "NFLX", "GOOG"]
+  const assets = ["AAPL", "BTC", "NFLX", "GOOG", "ETH", "SP500"]
```

Comment:
The underline below the assets needs to adapt to the current text, maybe we could implement that first 
<img width="486" alt="Screen Shot 2021-01-24 at 21 33 10" src="https://user-images.githubusercontent.com/9826683/105642750-e3c90800-5e8b-11eb-8739-1bc51d38da73.png">

Reply:
implemented that, should be good to go